tests/test_packages/test_skills/test_tac_negotiation/test_logical.py

```diff
@@ -0,0 +1,197 @@
+# -*- coding: utf-8 -*-
```

Comment:
These tests ensue logical behaviour of pricing (currently the functions ensure proposed prices actually increase the score)